app/src/main/res/layout/user_id_error.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/more_options_list_item_height">
```

Comment:
Can we have this height be reflected by its contents and use `wrap_content`? It's usually safer, in cases where the content needs to be larger than a fixed height.